server/libs/userlibs.js

```diff
@@ -252,4 +253,39 @@ const changePassword = function(data) {
     })
 }
 
-module.exports = {register, logIn, updateUser, logOut , changePassword, deleteUser, deActivateUser, getUser}
\ No newline at end of file
+const uploadPhoto = function(req) {
+    let id = req.params.uuid;
+    let form = new formidable.IncomingForm();
+    form.uploadDir = "public/images";
```

Comment:
where that image will be uploaded. are we saving the whole image in db or just a link

Reply:
the link should b in constant or json file

Reply:
Done